modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -1042,4 +1042,40 @@ a|The query used a procedure that generated a warning.
 m|WARNING
 |Category
 m|GENERIC
-|===
\ No newline at end of file
+|===
+
+[#_neo_clientnotification_statement_repeatedrelationshipreference]
+=== Neo.ClientNotification.Statement.RepeatedRelationshipReference (when run on version 5.5 or newer)
+
+.Notification category details
+[cols="<1s,<4"]
+|===
+|Code
+m|Neo.ClientNotification.Statement.RepeatedRelationshipReference
+|Title
+a|The query returns no results because a relationship variable is bound more than once.
+|Severity
+m|WARNING
+|Category
+m|GENERIC
+|===
+
+.Binding a relationship variable more than once.
+====
+Query::
++
+[source,cypher]
+----
+MATCH (:A)-[r]->(), ()-[r]->(:B) RETURN r
+----
+Description of the returned code::
+A relationship is referenced more than once in the query, which leads to no results because relationships must not occur more than once in each result. (Relationship `r` was repeated)
```

Comment:
```suggestion
A relationship is referenced more than once in the query, which leads to no results because relationships must not occur more than once in each result (the repeated relationship variable is `r`).
```

Reply:
This is the existing description, so let's not update it.

Reply:
Like Linnéa said, this is already in product as of 5.4.